libs/SalesforceSDKCore/SalesforceSDKCore/Classes/RestAPI/SFRestAPI.h

```diff
@@ -166,6 +166,53 @@ NS_SWIFT_NAME(RestClient)
  */
 - (SFRestRequest *)cheapRequest:(nullable NSString *)apiVersion;
 
+/**
+ * Returns an `SFRestRequest` object to retrieve the available notification types.
+ * The notification types define the possible push notifications the app can handle,
+ * including their structure and associated actions.
+ *
+ * This method automatically uses the default API version.
+ *
+ * @return An `SFRestRequest` object for fetching the notification types.
+ */
+- (SFRestRequest *)requestForNotificationTypes;
```

Comment:
We have a notifications category for RestAPI, wondering if these methods should move there too? https://github.com/forcedotcom/SalesforceMobileSDK-iOS/blob/dev/libs/SalesforceSDKCore/SalesforceSDKCore/Classes/RestAPI/SFRestAPI%2BNotifications.h

Reply:
That makes sense. Sorry @Crebs I sent you to the wrong file.

Reply:
ah, yeah.  I see @bbirman commented that in Slack too.  I'll move those around. Thanks for keeping me honest 👍 .